sslcontext-kickstart/src/main/java/nl/altindag/ssl/util/SSLFactoryUtils.java

```diff
@@ -50,4 +55,20 @@ private static <T> void reload(SSLFactory baseSslFactory,
         }
     }
 
+    public static void configureAsDefault(SSLFactory sslFactory) {
```

Comment:
Method serves original request from the issue #250, but it's misleading.
Looking at location (SSLFactoryUtils) and name I would assume it sets entire SSLContext as default.
I see 2 options:
- move it to TrustManagerUtils
- Create RootSSLContextSpi similar to RootTrustManagerSpi

Reply:
There is also third option change the name of the method

Reply:
Sounds good, however I discovered that the RootKeymanagerFactorySpi is not working as well as the SSLContextSpi. The SSLContextSpi I have not comitted yet. Do you have working code examples for those two?